python/chronos/src/bigdl/chronos/model/tf2/Seq2Seq_keras.py

```diff
@@ -143,6 +143,22 @@ def from_config(cls, config):
 
 
 def model_creator(config):
+    model = LSTMSeq2Seq(input_feature_num=config["input_feature_num"],
+                        output_feature_num=config["output_feature_num"],
+                        future_seq_len=config["future_seq_len"],
+                        lstm_hidden_dim=config.get("lstm_hidden_dim", 128),
+                        lstm_layer_num=config.get("lstm_layer_num", 2),
+                        dropout=config.get("dropout", 0.25),
+                        teacher_forcing=config.get("teacher_forcing", False))
+    learning_rate = config.get('lr', 1e-3)
+    model.compile(optimizer=getattr(tf.keras.optimizers,
+                                    config.get("optim", "Adam"))(learning_rate),
+                  loss=config.get("loss", "mse"),
+                  metrics=[config.get("metics", "mse")])
+    return model
+
+
+def model_creator_auto(config):
```

Comment:
add comment here to state why we need this special model creator.

Reply:
The reason is as follows:
The new **model_creator_auto** function adds two lines of code compared to the original **model_creator**, which will run (model(inputs)) to initialize the weights when creating the model, which is required for the AutoSeq2Seq model.

Reply:
yeah just put a simple version to the src's comment. e.g. add model(inputs) in this model creator to initialize the weights

Reply:
got it

Reply:
great, will merge once the pr validations pass